collections.md

```diff
@@ -344,10 +352,11 @@ You may even pass a callback as the default value. The result of the callback wi
 <a name="method-groupby"></a>
 #### `groupBy()` {#collection-method}
```

Comment:
Unrelated to this PR, but why do all of these have `groupBy()`, or equiv. That's not the function definition. Surely we should ditch the brackets, or actually write the function definition down?